CMakeLists.txt

```diff
@@ -86,11 +87,21 @@ option(OCIO_WARNING_AS_ERROR "Set build error level for CI testing" OFF)
 
 if(OCIO_BUILD_GPU_TESTS OR OCIO_BUILD_APPS)
     set(OCIO_GL_ENABLED ON)
-    find_package(OpenGL QUIET)
-    if(NOT OpenGL_FOUND)
-        message(WARNING "OpenGL not found.")
+
+    find_package(OpenGL REQUIRED COMPONENTS OpenGL)
+    if(NOT OpenGL_OpenGL_FOUND AND NOT OPENGL_GLU_FOUND)
+        message(WARNING "OpenGL not found. GPU rendering disabled.")
         set(OCIO_GL_ENABLED OFF)
     endif()
+    # OpenGL_egl_Library is defined iff GLVND is supported (CMake 10+)
```

Comment:
There are two distinct blocks of code here so you should add  an empty line between line 95 and 96.